README.md

```diff
@@ -90,3 +90,6 @@ the changes were fundamental enough that the decision was made to create soci-sn
 project.  Soci-snapshotter builds on stargz's success and innovative ideas.  Long term, this project intends
 and hopes to join [containerd](https://github.com/containerd/containerd) as a non-core project and intends to
 follow CNCF best practices.
+
+## Terminology
```

Comment:
If getting started ever gets added this section would be slightly different as I've changed the README to include a learning more section